cesm2_albedo/README

```diff
@@ -0,0 +1,75 @@
+This data file can be downloaded from [LLNL](https://aims3.llnl.gov/thredds/fileServer/css03_data/CMIP6/CMIP/NCAR/CESM2/historical/r1i1p1f1/Amon/rsds/gn/v20190308/rsds_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc
+)
+
+### 2. `rsus_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc`
+
+This netCDF file contains Surface Downwelling Shortwave Radiation in the variable `rsds`.
+This variables is defined on the longitude, latitude, and time dimensions. `rsds` is taken as
+the montly average, with the first data point at 15/01/1850 12:00 and the las at 15/12/2014 12:00.
+This data file can be downloaded from [LLNL](https://aims3.llnl.gov/thredds/fileServer/css03_data/CMIP6/CMIP/NCAR/CESM2/historical/r1i1p1f1/Amon/rsus/gn/v20190308/rsus_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc
+)
+
+## Output Files
+
+### 1. `sw_albedo_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412_v2_no-nans.nc`
+
+This netCDF file contains shortwave albedo in the variable `sw_alb`. The variable is defined
+on the longitude, latitude, and time dimensions. `sw_alb` is taken as the montly average,
+with the first data point at 15/01/1850 12:00 and the las at 15/12/2014 12:00. `sw_alb` is
+calculated by dividing Surface Upwelling Shortwave Radiation by Surface Downwelling Shortwave Radiation.
+At any point, if Surface Downwelling Shortwave Radiation is zero, `sw_alb` is set to 1.0.
+`sw_alb` is stored as a Float32.
+
+### 2. `bareground_albedo.nc`
+
+This netCDF file contains average shortwave albedo in the summer for each hemisphere.
+The variable is defined on the longitude and latitude. In the northern hemisphere the average
+is taken of shortwave albedo during May, June, and July. In the southern hemisphere the average
+is taken of shortwave albedo during December and January.
+At any point, if data is missing, `sw_alb` is set to 1.0.
+`sw_alb` is stored as a Float32.
+
+## Scripts
+
+### `calculate_sw_alb.jl`
+
+This script processes `rsds_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc` and
+`rsus_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc` to create
+`sw_albedo_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412_v2_no-nans.nc`. This is done by
+dividing `rsus` by `rsds`, and assigning a value of 1.0 when `rsds` is zero.
+
+### `calculate_bareground_alb.jl`
+
+This script processes `sw_albedo_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412_v2_no-nans.nc`
+to create `bareground_albedo.nc` This is done by averaging the input `sw_alb` during the
+summer months for each hemisphere. To do this, first mean albedo is calulated globally
+for the northern hemisphere summer months. Then, mean albedo is calulated globally
+for the southern hemisphere summer months. Then, these two mean albedo's are merged, with the
+northern hemisphere set to the average during the northen hemisphere summer months and the
+southern hemisphere set to the average during the southern hemisphere summer months
+
+## References
+
+## License
+
```

Comment:
Who would these or know where I can get these?

Reply:
I found [this page](https://www.ucar.edu/terms-of-use/data) of the UCAR website which says they use the [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/legalcode)